### PR TITLE
Permission fixes for GraderUser

### PR DESCRIPTION
**Pull request by lainets**
# Description

**What?**

Fixes GraderUser permissions related to Jutut access.

**Why?**

Jutut either couldn't access exercises or it crashed A+.

**How?**

Check that the user is not a GraderUser in normal permissions, and add exercise read permission to exercise tokens.

# Testing

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [X] Manual testing.

[ADD A DESCRIPTION ABOUT WHAT YOU TESTED MANUALLY]

**Did you test the changes in**

- [ ] Chrome
- [ ] Firefox
- [X] This pull request cannot be tested in the browser.

**Think of what is affected by these changes and could become broken**

# Programming style

- [X] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

File: course/permissions.py [hunk 1]
```diff
@@ -130,6 +130,9 @@ def is_object_visible(self, request, view, course):
         """
         # NOTE: course is actually course instance
 
+        if isinstance(request.user, GraderUser):
```
Comment (Mankro) [inline course/permissions.py:133]: What has changed compared to A+ v1.11? I assume that GraderUser was never before using this permission, so why is it now here?

**Review by Mankro — APPROVED** [review]
Good, thanks! I can merge this now. I had one question which I would still like to understand better,

File: course/permissions.py (hunk 1 above)
Comment (lainets) [reply course/permissions.py:133]: It was using it, the GraderUser was just always assumed to be a teacher, so it got skipped on the next if statement.

Comment (Mankro) [reply course/permissions.py:133]: So what happened if GraderUser is now not counted as a teacher?

Comment (lainets) [reply course/permissions.py:133]: Being a teacher implies write permissions to a course, so we do not assume that the GraderUser is always a teacher. They are only counted as a teacher if they have write permissions to the course (listed in the JWT)

Comment (Mankro) [reply course/permissions.py:133]: Thanks for the clarification!

**Pull request merged**